pages/agent/v3/securing.md.erb

```diff
@@ -45,6 +45,10 @@ Command line evaluation can be disabled by setting [`no-command-eval`](/docs/age
 
 As plugins execute in the same way as local hooks, they can pose a potential security risk. If you're using third party plugins, you'll be executing the third party's code on your agent.
 
+You can disable plugins with the command line flag: `--no-plugins`.  
+
+If you still would like to use plugins, a third-party tool for signing pipelines will help you do it securely. See [Signing pipelines](https://buildkite.com/docs/agent/v3/securing#signing-pipelines) for more information.  
```

Comment:
More direct is better. :-)
And links without buildkite.com make it much easier to review PRs, because the links in previews work.

```suggestion
If you still want to use plugins, a third-party tool for signing pipelines will help you do it securely. See [Signing pipelines](/docs/agent/v3/securing#signing-pipelines) for more information.  
```

Reply:
Oops. Done.